SPECS/kernel-mos/kernel-mos.spec

```diff
@@ -18,7 +18,7 @@
 %define config_source %{SOURCE1}
 Summary:        Linux Kernel for MOS
 Name:           kernel-mos
-Version:        5.15.158.1
+Version:        5.15.158.2
```

Comment:
The linked buddy build seems to have failed.

Reply:
It seems to require a minor kernel config tweak. From the logs:
```
time="2024-06-07T18:48:28Z" level=debug msg="--- new_config\t2024-06-07 18:48:28.613018149 +0000"
time="2024-06-07T18:48:28Z" level=debug msg="+++ current_config\t2024-06-07 18:48:25.689053082 +0000"
time="2024-06-07T18:48:28Z" level=debug msg="@@ -1,6 +1,6 @@"
time="2024-06-07T18:48:28Z" level=debug msg=" #"
time="2024-06-07T18:48:28Z" level=debug msg=" # Automatically generated file; DO NOT EDIT."
time="2024-06-07T18:48:28Z" level=debug msg="-# Linux/x86_64 5.15.158.2 Kernel Configuration"
time="2024-06-07T18:48:28Z" level=debug msg="+# Linux/x86_64 5.15.158.1 Kernel Configuration"
time="2024-06-07T18:48:28Z" level=debug msg=" #"
time="2024-06-07T18:48:28Z" level=debug msg=" CONFIG_CC_VERSION_TEXT=\"gcc (GCC) 11.2.0\""
time="2024-06-07T18:48:28Z" level=debug msg=" CONFIG_CC_IS_GCC=y"
time="2024-06-07T18:48:28Z" level=debug
time="2024-06-07T18:48:28Z" level=debug
time="2024-06-07T18:48:28Z" level=debug
time="2024-06-07T18:48:28Z" level=debug
time="2024-06-07T18:48:28Z" level=debug
time="2024-06-07T18:48:28Z" level=debug
time="2024-06-07T18:48:28Z" level=debug
time="2024-06-07T18:48:28Z" level=debug
time="2024-06-07T18:48:28Z" level=debug msg="Config file has unexpected changes"
```

Reply:
yep, just found and fixed that, thanks